src/server/conf/passport.js

```diff
@@ -14,5 +16,8 @@ module.exports = {
             clientSecret: process.env.FB_CLIENT_SECRET,
             callbackURL: process.env.FB_CALLBACK_URL || `${appConf.protocol}://${appConf.host}:${appConf.port}/auth/fb/callback`,
         },
+        local: {
+            hashPassword: password => md5(password + process.env.SALT),
```

Comment:
Any variable that is stored in `process.env` object can be set by 2 ways:
- on script start-up. Example `VAR_NAME="var_value" node script.js`
- in OS defaults. It'll be static, until somebody change it.

As I think it's a good way to let anyone contribute here without any SALT value and pretty secure way to hide real password of real users.